### PR TITLE
Button styling

### PR DESCRIPTION
**Pull request by trbachmann**
* Added react-tooltip - reminder run `npm install` because I installed it as a dependency

* Added custom tooltips when hovering over visited and bucket list icons. Text for tooltips change if they have the park saved/or not on their list
* Updated styling for visited and bucket list icons to match the map icon colors
* Updated styling for delete icon
* Added a method that comma separates the annual visitors number on each park card

### CODE REVIEW

File: src/Buttons.js [hunk 1]
```diff
@@ -35,25 +36,31 @@ class Buttons extends Component {
...
     this.props.updateParkCodes(storageKey, newArray);
   }
-  
+
   render() {
     let buttonType;
+    let toolTipText;
     if (this.state.storageKey === 'visitedParks') {
       buttonType = this.props.visitedParks;
+      toolTipText = buttonType.includes(this.state.parkUrl) ? 'Remove from Visited Parks' : 'Add to Visited Parks';
```
Comment (kaohman) [inline src/Buttons.js:45]: Nice use of ternaries here!

File: src/Park.js [hunk 2]
```diff
@@ -18,16 +18,23 @@ class Park extends Component {
...
+      annualVisitors: 0
     });
   }
 
   updateParkCodes = (storageKey, newArray) => {
     this.props.updateParkCodes(storageKey, newArray);
   }
 
+  commaSeparateAnnualVisitors = () => {
+    let visitors = parseInt(this.props.selectedPark.annualVisitors);
```
Comment (kaohman) [inline src/Park.js]: Is this line necessary? I thought annual visitors was stored as a number in the dataset so we may not need this step.

File: src/Park.js [hunk 3]
```diff
@@ -18,16 +18,23 @@ class Park extends Component {
...
+    return visitors.toLocaleString();
```
Comment (kaohman) [inline src/Park.js]: Awesome! I has no idea this prototype existed.

**Review by kaohman — APPROVED** [review]
Looks great, I especially like your use of ternary operators for the tooltips.

File: src/Park.js (hunk 2 above)
Comment (trbachmann) [reply src/Park.js]: Good catch thank you! I thought it was saved as a string, but you're right it is stored as a number. I'm going to push up changes with this removed. thanks!

**Comment by trbachmann** [comment]
@kaohman thank you! Good catch on the annual visitors being stored as a number. For some reason I thought it was stored as a string. I updated Park.js to remove that and was able to convert the annual visitors to be comma separated without the method now and just using toLocaleString().

**Pull request merged**